seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_to_assert.conf

```diff
@@ -51,22 +66,7 @@ source {
           c_date = date
           c_decimal = "decimal(38, 18)"
           c_timestamp = timestamp
-          c_row = {
```

Comment:
Why delete it?

Reply:
@TyrantLucifer  It's not deletion, but not using the hecon plug-in to format the file content at that time.